### PR TITLE
feat: formatters support SVG

### PR DESCRIPTION
**Pull request by yunsii**
### Description

- @prettier/plugin-xml support [formatting SVG](https://github.com/prettier/plugin-xml/issues/27) and I tried, it works like a charm

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -93,6 +105,11 @@ export interface OptionsFormatters {
    */
   prettierOptions?: VendoredPrettierOptions
 
+  /**
+   * Custom overrides let you have different configuration for certain file extensions and specific files.
+   */
+  prettierOverrides?: (options: PrettierOverridesOptions) => TypedFlatConfigItem[]
```
Comment (antfu) [inline src/types.ts:111]: I am not sure if I understand the purpose of this hook. I'd say we better move it to a separate PR so we can move the SVG support forward faster

Comment (yunsii) [reply src/types.ts]: OK, I will create a new PR for `prettierOverrides`.

File: src/configs/formatters.ts [hunk 2]
```diff
@@ -10,22 +10,24 @@ export async function formatters(
...
+      xml: isPrettierPluginXmlInScope,
     }
   }
 
   await ensurePackages([
     'eslint-plugin-format',
     options.markdown && options.slidev ? 'prettier-plugin-slidev' : undefined,
     options.astro ? 'prettier-plugin-astro' : undefined,
-    options.xml ? '@prettier/plugin-xml' : undefined,
+    options.xml || options.svg ? '@prettier/plugin-xml' : undefined,
```
Comment (antfu) [inline src/configs/formatters.ts:30]:
```suggestion
    (options.xml || options.svg) ? '@prettier/plugin-xml' : undefined,
```

**Review by antfu — APPROVED** [review]
